src/data/countable/basic.lean

```diff
@@ -0,0 +1,108 @@
+/-
+Copyright (c) 2022 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury Kudryashov
+-/
+import logic.equiv.nat
+import logic.equiv.fin
+import data.countable.defs
+
+/-!
+# Countable `Sort*`s and `Type*`s
+
+In this file we define a typeclass saying that a given `Sort*` is countable. See also `encodable`
```

Comment:
Isn't this rather defined in `data.countable.defs`?

Reply:
That's why this PR has the label `needs-documentation`.